### PR TITLE
handle case where no runtime_envs are defined

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/support/issues/171

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

create default case for when no runtime environments are defined in .thoth.yaml

### CODE REVIEW

File: kebechet/managers/update/update.py [hunk 1]
```diff
@@ -797,16 +805,21 @@ def run(self, labels: list) -> Optional[dict]:
             )
 
             for e in runtime_environments:
-                self.runtime_environment = e
+                self.runtime_environment = e or "default"
```
Comment (harshad16) [inline kebechet/managers/update/update.py:808]: is the default here suppose to be "."?

File: kebechet/managers/update/update.py [hunk 2]
```diff
@@ -797,16 +805,21 @@ def run(self, labels: list) -> Optional[dict]:
...
                 close_no_management_issue = partial(
                     self.close_issue_and_comment,
-                    _ISSUE_NO_DEPENDENCY_NAME.format(env_name=e),
+                    _ISSUE_NO_DEPENDENCY_NAME.format(env_name=self.runtime_environment),
                     comment=ISSUE_CLOSE_COMMENT.format(sha=self.sha),
                 )
-                with cwd(thoth_config.get_overlays_directory(e)):
+                env_dir = thoth_config.get_overlays_directory(e) if e else "."
```
Comment (harshad16) [inline kebechet/managers/update/update.py:814]: is the `e` here intentional ? should this be `self.runtime_environment`

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
just few doubts?

File: kebechet/managers/update/update.py (hunk 1 above)
Comment (KPostOffice) [reply kebechet/managers/update/update.py:808]: `self.runtime_environment` is used to create issue contents so I opted for something human readable

File: kebechet/managers/update/update.py (hunk 2 above)
Comment (KPostOffice) [reply kebechet/managers/update/update.py:814]: This is on purpose, self.runtime_environment will always be set now but e still won't be.

**Comment by KPostOffice** [comment]
@harshad16 PTAL

**Comment by goern** [comment]
is this ready for approval?

**Comment by goern** [comment]
is this good to be merged?

**Comment by KPostOffice** [comment]
@goern Should be good 2 go

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :100:

**Pull request merged**